Components/Coroutines/MoveToAndInteract.cs

```diff
@@ -35,8 +35,7 @@ public static async Task<bool> Execute(DiaObject obj, int interactLimit = 5)
             if (Core.Player.IsInTown)
                 GameUI.CloseVendorWindow();
 
-            return await CommonCoroutines.MoveAndInteract(obj, () => !IsInteracting &&
-                                                                     interactLimit-- > 0) == CoroutineResult.Running;
+            return await CommonCoroutines.MoveAndInteract(obj, () => IsInteracting || interactLimit-- < 0) == CoroutineResult.Running;

```

Comment:
this should be `interactLimit-- <= 0` ?

Reply:
I also feel like the substitution here might cause other issues. Not sure how and when the delegate is getting evaluated but there might be cases that this gets evaluated more then intended.